db/migrate/20181207011115_downcase_custom_emoji_domains.rb

```diff
@@ -0,0 +1,5 @@
+class DowncaseCustomEmojiDomains < ActiveRecord::Migration[5.2]
```

Comment:
Add `disable_ddl_transaction!` to use no locking around the updates.

Reply:
Thanks, I'm learning a lot about ActiveRecord migrations :)